aws/resource_aws_route_table_test.go

```diff
@@ -377,6 +377,30 @@ func TestAccAWSRouteTable_Route_TransitGatewayID(t *testing.T) {
 	})
 }
 
+func TestAccAWSRouteTable_Route_LocalGatewayID(t *testing.T) {
+	var routeTable1 ec2.RouteTable
+	resourceName := "aws_route_table.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
This will allow test environments without an available Outpost to skip the test instead of returning an error:

```suggestion
		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSOutpostsOutposts(t) },
```